versions.json

```diff
@@ -1,13 +1,13 @@
 [
     {
-        "neo4j": "4.4.4",
-        "neo4jVersion": "4.4.4",
-        "apoc": "4.4.0.3",
-        "version": "4.4.0.3",
-        "url": "http://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/4.4.0.3",
-        "homepageUrl": "http://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/4.4.0.3",
-        "jar": "https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/4.4.0.3/apoc-4.4.0.3-all.jar",
-        "downloadUrl": "https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/4.4.0.3/apoc-4.4.0.3-all.jar",
+        "neo4j": "5.0.0",
+        "neo4jVersion": "5.0.0",
+        "apoc": "5.0.0",
+        "version": "5.0.0",
+        "url": "http://github.com/neo4j/apoc/releases/5.0.0",
+        "homepageUrl": "http://github.com/neo4j/apoc/releases/5.0.0",
+        "jar": "https://github.com/neo4j/apoc/releases/download/5.0.0/apoc-5.0.0.jar",
+        "downloadUrl": "https://github.com/neo4j/apoc/releases/download/5.0.0/apoc-5.0.0.jar",
```

Comment:
We should use https://github.com/neo4j/apoc/releases/download/5.0.0-rc02/apoc-5.0.0-rc02-core.jar, which is a real version here